### PR TITLE
fix(schemas/yazi.json,theme.json,keymap.json): replaced `[select]` with `[pick]` for Yazi@0.4.0

### PR DESCRIPTION
**Pull request by lamteteeow**
Resolved breaking changes when migrating to Yazi@0.4.0 mentioned on
[Issue#1772](https://github.com/sxyazi/yazi/issues/1772) section 1.2.

### CODE REVIEW

File: schemas/keymap.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
 			},
 			"additionalProperties": false
 		},
-		"select": {
```
Comment (lamteteeow) [inline schemas/keymap.json:29]: Based on [Issue#1772](https://github.com/sxyazi/yazi/issues/1772) section 1.2

File: schemas/theme.json [hunk 2]
```diff
@@ -73,7 +73,7 @@
 			},
 			"additionalProperties": false
 		},
-		"select": {
```
Comment (lamteteeow) [inline schemas/theme.json:76]: Based on [Issue#1772](https://github.com/sxyazi/yazi/issues/1772) section 1.2

File: schemas/yazi.json [hunk 3]
```diff
@@ -250,7 +250,7 @@
 			},
 			"additionalProperties": false
 		},
-		"select": {
```
Comment (lamteteeow) [inline schemas/yazi.json:253]: Based on [Issue#1772](https://github.com/sxyazi/yazi/issues/1772) section 1.2

**Review by lamteteeow — COMMENTED** [review]
Based on [Issue#1772](https://github.com/sxyazi/yazi/issues/1772) section 1.2

**Comment by uncenter** [comment]
See https://github.com/yazi-rs/schemas/blob/main/CONTRIBUTING.md#updating-for-a-new-yazi-release.

**Comment by lamteteeow** [comment]
> See https://github.com/yazi-rs/schemas/blob/main/CONTRIBUTING.md#updating-for-a-new-yazi-release.

Thank you, ill check it out. Apologize as I am new to the whole open source thing. Sorry again for any inconvenience earlier or in the future.

**Pull request closed without merging**